index.html

```diff
@@ -28,7 +37,7 @@
         <div class="intro">
             
             <div class="col-12 ">
-                <h1 class="col-h col-11 col-lg-5 aligne-self-center">I'm Caleb Moses </h1>
+                <h1 class="col-h col-11 col-lg-5 aligne-self-center">I'm Moses </h1>
                 <h1 class="col-h col-11 col-lg-5 aligne-self-center"> glade to see you</h1>
                 <p class="col col-12">I’m a software developer! I can help you build a product ,
                     feature or website Look through some of my work and experience! 
```

Comment:
You've created a wonderful headling :handshake: 
But since we have to ensure that your design is similar to the Figma design, I think you should consider increasing the font size of your headline. It's a bit too tiny at the moment and I think increasing it will make the headline look better. Please check the images below to get my point better
**_Figma Design_**
![original](https://user-images.githubusercontent.com/26856491/121506626-caa89a00-c9db-11eb-9fc9-20646a492a38.png)

_**Your Design**_
![clone](https://user-images.githubusercontent.com/26856491/121506639-ced4b780-c9db-11eb-87ea-67a1af9d7aac.png)

Reply:
Did this!👍